corehq/apps/callcenter/sync_usercase.py

```diff
@@ -93,18 +93,17 @@ def _domain_has_new_fields(domain, field_names):
     return False
 
 
-def _get_sync_usercase_helper(commcare_user, case_type, owner_id, case=None):
-    domain = commcare_user.domain
-    fields = _get_user_case_fields(commcare_user, case_type, owner_id)
-    case = case or CaseAccessors(domain).get_case_by_domain_hq_user_id(commcare_user._id, case_type)
-    close = commcare_user.to_be_deleted() or not commcare_user.is_active
-    user_case_helper = _UserCaseHelper(domain, owner_id, commcare_user._id)
+def _get_sync_usercase_helper(user, domain_obj, case_type, owner_id, case=None):
```

Comment:
Agree with this, just wanted to show that it's fine to re-fetch domain objects all over the place, because we cache `Domain.get_by_name`: https://github.com/dimagi/commcare-hq/blob/fc8ca8faa838f5bbee50d164147e5b3078fbfe41/corehq/apps/domain/models.py#L607-L610